egov-indexer/open-hearing-indexer.yml

```
@@ -34,4 +34,8 @@ ServiceMaps:
               - inJsonPath: $.fromDate
                 outJsonPath: $.Data.hearingDetails.fromDate
               - inJsonPath: $.toDate
-                outJsonPath: $.Data.hearingDetails.toDate
\ No newline at end of file
+                outJsonPath: $.Data.hearingDetails.toDate
+              - inJsonPath: $.subStage
+                outJsonPath: $.Data.hearingDetails.subStage
+              - inJsonPath: $.courtId
+                outJsonPath: $.Data.hearingDetails.courtId
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)